bcda/api/requests.go

```diff
@@ -62,13 +64,21 @@ func NewHandler(resources []string, basePath string) *Handler {
 	cutoffDuration := time.Duration(utils.GetEnvInt("CCLF_CUTOFF_DATE_DAYS", 45)*24) * time.Hour
 
 	// Allow runout requests to be up to 4 months after runout data was ingested
-	runoutCutoffDuration := time.Duration(utils.GetEnvInt("RUNOUT_CUTOFF_DATE_DAYS", 45)*24) * time.Hour
+	runoutCutoffDuration := time.Duration(utils.GetEnvInt("RUNOUT_CUTOFF_DATE_DAYS", 120)*24) * time.Hour
+	runoutClaimThru := utils.FromEnv("RUNOUT_CLAIM_THRU_DATE", "2020-12-31")
```

Comment:
How do we ensure we remember to set this to `2021-12-31` at the end of next year?

Reply:
Good call out! I added a scheduled trigger to update on 2021-11-15:

https://jira.cms.gov/projects/BCDA?selectedItem=pl.com.tt.jira.plugin.theschedulerpro:scheduled-issues-project-tab-panel-link&scheduledJobId=236